static/sass/_v1_pattern_navigation.scss

```diff
@@ -0,0 +1,198 @@
+@mixin p-navigation {
+
+  html {
+    overflow-x: hidden !important; // prevent horizontal overflow full width breakout
+  }
+
+  .p-navigation {
+    margin-bottom: 0;
+    border-bottom: 1px solid $color-light;
+
+    &__row {
+      @media (min-width: $breakpoint-large) {
+        position: relative;
+      }
+    }
+  }
+
+  .p-navigation__links .p-navigation {
+    background-color: $color-mid-dark;
+
+    &__link {
+
+      &:hover {
+        cursor: pointer;
+      }
+
+      @media (max-width: $breakpoint-medium - 1) {
+        width: 50%;
+        float: left;
+        padding: $sp-x-small $sp-small;
+
+        &:nth-child(2n) {
+          border-left: 1px solid $color-mid-light;
+        }
+
+        &:last-of-type {
+          border-bottom: 1px solid $color-mid-light;
+        }
+      }
+
+      @media (min-width: $breakpoint-medium) {
+        padding: $sp-medium $sp-x-small;
+
+        &:hover {
+          background-color: $color-brand-light;
+        }
+      }
+
+      @media (min-width: $breakpoint-large) {
+        padding: $sp-medium $sp-small;
+      }
+    }
+  }
+
+  // .nav-secondary override
+  .nav-secondary {
+    // small screen view
+    @media (max-width: $breakpoint-medium - 1) {
+      background: $color-x-light;
+
+      .p-breadcrumbs {
+        @include full-width-breakout;
+        border-bottom: 1px solid $color-mid-light;
+      }
+
+      .p-breadcrumbs__item {
+        position: relative;
+        left: 50%;
+        transform: translate(-50%, 0);
+        width: 100vw;
+      }
+
+      .p-breadcrumbs__link {
+        border-bottom: 1px solid $color-mid-light;
+        width: 100%;
+        display: inline-block;
+        padding: $sp-x-small;
+        margin-bottom: $sp-medium;
+        color: $color-dark;
+      }
+
+      .p-inline-list {
+        margin-left: 0;
```

Comment:
`.p-inline-list` already has this rule applied, perhaps this is superfluous - removing.